### PR TITLE
Event management/database

### PR DESCRIPTION
**Pull request by Veaxx**
# Description

Please include a summary of the change, which issue is fixed and the context if the new feature. List any dependencies that are required for this change.

## Type of change

Please delete options that are not relevant and add new options if necessary. 

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings

### CODE REVIEW

**Review by BjornVanBruchem — APPROVED** [review]
Vraag me wel 'n beetje af of einddatum echt nodig is, maar voor nu goed

File: backend/seeds/eventSeeder.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+const mongoose = require('mongoose');
+
+const Events = mongoose.model('Event');
+
+const eventData = [
+  {
+    title: 'Pronkzitting',
+    description: 'Een jaarlijks terugkerend avondvullend programma waar het publiek door Schaijkse rasartiesten vermaakt wordt. De artiesten zijn Schaijkse mensen die met eigen teksten of eigen liedjes Schaijkse gebeurtenissen op de korrel nemen en het publiek weten te boeien. De voorbereidingen nemen lang van te voren hun aanvang, zo ongeveer bij de opening van het jaarlijkse carnaval, zijnde het 11-11 bal. Op de eerste bijeenkomst met artiesten en commissie wordt de opzet doorgesproken en gekeken wie er mee wil doen en wat voor ideeën er zijn.',
+    startdate: Date(),
```
Comment (xandermonk) [inline backend/seeds/eventSeeder.js:9]: Are all seed dates just the current date? Better to put an actual date here

File: backend/seeds/eventSeeder.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+const mongoose = require('mongoose');
+const Events = mongoose.model('Event');
+
+const today = new Date();
+const tomorrow = new Date(today);
+tomorrow.setDate(tomorrow.getDate() + 1);
+const dayAfterTomorrow = new Date(today);
+dayAfterTomorrow.setDate(dayAfterTomorrow.getDate() + 2);
+const dayAfterTheDayAfterTomorrow = new Date(today);
+dayAfterTheDayAfterTomorrow.setDate(dayAfterTheDayAfterTomorrow.getDate() + 3);
```
Comment (xandermonk) [inline backend/seeds/eventSeeder.js:10]:
```suggestion
function getDate(daysToAdd) {
  const today = new Date();
  const futureDate = new Date(today.getFullYear(), today.getMonth(), today.getDate() + daysToAdd);
  return futureDate;
}

const today = new Date();
const tomorrow = getDate(1);
const dayAfterTomorrow = getDate(2);
const dayAfterTheDayAfterTomorrow = getDate(3);
```

**Review by xandermonk — APPROVED** [review]

File: backend/models/event.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    startdate: {
+        type: Date,
+        required: true
+    },
+    enddate: {
+        type: Date,
+        required: true
+    },
+    location: {
+        type: String,
```
Comment (KevinVanH) [inline backend/models/event.js:21]: Moeten we van Location misschien ook niet een model maken? Kan me voorstellen dat een locatie een naam heeft, adres, coordinaten op een kaart etc

**Review by KevinVanH — APPROVED** [review]

**Pull request merged**